### PR TITLE
[CI] Skip CI when only docs are modified

### PR DESCRIPTION
**Pull request by kezhenxu94**
Skip CI when only docs are modified

### CODE REVIEW

File: tools/ci/changed.sh [hunk 1]
```diff
@@ -18,12 +18,18 @@
 
 patterns=()
 any_of=0
+all_of=0
 
+# --any-of: requires any of the changed files matche the given patterns
+# --all-of: requires all of the changed files matche the given patterns
```
Comment (wu-sheng) [inline tools/ci/changed.sh]: Is this the right name? From your usage, if the all-of matched, the job is not working. But for the any-of, if  matched, the job runs. Seems different logic.

Comment (wu-sheng) [reply tools/ci/changed.sh]: exclusion seems a better name?

Comment (kezhenxu94) [reply tools/ci/changed.sh]: this script is nothing related to the Jenkins logic from the file name `tools/ci/changed.sh`, it just determines whether (all of/any of) the files matching the given patterns changed or not, the usage sounds something like `do all of the changed files match the given patterns`, or `does any of the changed files match the given patterns`, `exclusion` seems also ok to me but may be less semantic

Comment (kezhenxu94) [reply tools/ci/changed.sh]: Jenkins just determine whether to run based on the result of this script

Comment (wu-sheng) [reply tools/ci/changed.sh]: Gor your point. It is ok to unchange.

**Pull request merged**